PD-1405 / 24.10 / PD-1405-the-static-ip-settings-from-console-are-inaccurate (#3034)
truenas/documentation: static/includes/AliasOrStaticIP.md

{{< expand "One or More Aliases?" "v" >}}
Static IP addresses set a fixed address for an interface that external devices or websites need to access or remember, such as for VPN access.

> Use aliases, also known as static IP addresses, to add multiple internal IP addresses, representing containers or applications hosted in a VM, to an existing network interface without having to define a separate network interface.

Reviewer: This is awkwardly worded and doesn't sound correct. how about this instead
```suggestion
Use aliases or static IP addresses to add multiple internal IP addresses that represent containers or applications hosted in a VM to an existing network interface without having to define a separate network interface.
```
